lib/msal-electron-proof-of-concept/README.md

```diff
@@ -0,0 +1,54 @@
+
+Microsoft Authentication Library for Electron Proof of Concept
+==============================================================
+
+The MSAL library for Electron enables cross-platform Electron desktop applications to authenticate users using [Azure AD](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-overview) work and school accounts (AAD) as well as Microsoft personal accounts (MSA). It also enables your Electron app to get tokens to access [Microsoft Cloud](https://cloud.microsoft.com) services such as [Microsoft Graph](https://graph.microsoft.io).
+
+## Installation
+
+Using NPM:
+
+```bash
+npm install msal-electron-poc
```

Comment:
`electron` needs to be a peer dep right? Worth including explicitly.

Reply:
Done, added a couple notes on the subject below.